### PR TITLE
Release 1.40.0

### PR DESCRIPTION
**Pull request by aaronfc**
Release branch.

## Includes
- Bump version to 1.40.0
- Bump minimum php version to 7.4
- Regenerate pot files.
- Regenerate changelogs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wp-job-manager",
-  "version": "1.39.0",
+  "version": "1.40.0",
```
Comment (renatho) [inline package.json:3]: I think we need to run an `npm install` to also update the `package-lock`.

File: changelog.txt [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # WP Job Manager
 
+## 1.40.0 - 2023-05-11
```
Comment (renatho) [inline changelog.txt:3]: Is it this date on purpose because we plan to release this date?

**Review by renatho — APPROVED** [review]
Just one small tweak and a question about the release date. 😉
I'm already approving it, but I think it's worth waiting for a double check from @Automattic/zen.

**Review by yscik — APPROVED** [review]
🙌

File: changelog.txt (hunk 2 above)
Comment (aaronfc) [reply changelog.txt:3]: That's right :)

File: package.json (hunk 1 above)
Comment (aaronfc) [reply package.json:3]: Done! [a28122f](https://github.com/Automattic/WP-Job-Manager/pull/2413/commits/a28122f7bc329835219f15d905c43f8d484730b4)

**Pull request merged**